File: 000. 两数之和 Two Sum/000.js
<<<<<<< HEAD

//@param {number[]} nums
//@param {number} target
//@return {number[]}
 
var twoSum = function(nums, target) {
    for (let i=0;i  nums.length; i++)
        for (let j=i+1; j  nums.length; j++)
            {
                if (nums[i]+nums[j]==target) return [i, j];
            }
=======

//@param {number[]} nums
//@param {number} target
//@return {number[]}
 
var twoSum = function(nums, target) {
    for (let i=0;i  nums.length; i++)
        for (let j=i+1; j  nums.length; j++)
            {
                if (nums[i]+nums[j]==target) return [i, j];
            }
>>>>>>> 89f709a1ed67f322bd9b0ae462cfe34ffaebd882
};

File: 000. 两数之和 Two Sum/000.cs
public class Solution
        {
            public int[] TwoSum(int[] nums, int target)
            {
                for (var i = 0; i < nums.Length; i++)
                    for (var j = i + 1; j < nums.Length; j++)
                    {
                        if (nums[i] + nums[j] == target)
                        {
                            var ans = new int[2] { i, j };
                            return ans;
                        }
                    }
                throw new Exception("No two sum solution");
            }
        }